### ISSUE TITLE
fastai usage question

### ISSUE THREAD

**Issue by micimize** [KevinMusgrave/pytorch-metric-learning#133]
I'm fairly green to ML, so this might be a naive question, but if I'm just wanting to use a mining strategy with fastai, is this all I need to do?
```python
def mined_loss(miner, loss_func):
    def loss(embeddings, labels):
      mined = miner(embeddings, labels)
      return loss_func(embeddings, labels, mined)
    return loss
  
my_loss = mined_loss(miner = miners.MultiSimilarityMiner(epsilon=0.1),
                     loss_func = losses.TripletMarginLoss(margin=0.1))
 
# for fastai
learn = Learner(data, model, loss_func = my_loss)
# etc, etc
```

I guess this maybe a side-effect of learning with higher-level abstractions first, is that it makes more generic usages seem opaque 😅

**Comment by KevinMusgrave** [comment]
Yeah that looks correct to me
